course-bundle-spec.md

```diff
@@ -0,0 +1,125 @@
+# Qwiklabs Course Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: Course
+schema_version: 1
+
+# Course Attributes
+id: gcp-intro-course
+default_locale: en
```

Comment:
Modality: my understanding is that we want courses in general to be usable both as Instructor-Led and Self-Paced. Do you think there should be an intended modality, but the other is always available? Or maybe two attributes: `instructor_led` and `self_paced` and people could set one or both of them?

Course Code: Can that just be the `id`?

Level: Oy, this again :-p. Yeah we could add one - would you expect it to be an enum, or freetext?

Reply:
We agreed:
- Modality will not be included, since we want the content to be used in either context
- People can make the `id` their course code if they want
- We will add level: 1, 2, 3, 4